docs/platform/howto/byoc/create-custom-cloud/create-google-custom-cloud.md

```diff
@@ -0,0 +1,491 @@
+---
+title: Create a Google-integrated custom cloud
+sidebar_label: Google Cloud
+keywords: [Google Cloud, GCP, Google Cloud Platform, byoc, bring your own cloud, custom cloud]
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Create a [custom cloud](/docs/platform/concepts/byoc) for BYOC in your Aiven organization to better address your specific business needs or project requirements.
+
+To configure a custom cloud in your Aiven organization and prepare your Google Cloud
+account so that Aiven can access it:
+
+1. In the Aiven Console or with the Aiven CLI client, you specify new cloud details to
+   generate a Terraform infrastructure-as-code template.
+1. You download the generated template and deploy it in your Google Cloud account to acquire
+   a privilege-bearing service account, which Aiven needs for accessing your Google
+   Cloud account only with permissions that are required.
+
+   :::note
+   Privilege-bearing service account is an
+   [identifier](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_service_account#id)
+   of the [service account](https://cloud.google.com/iam/docs/service-account-types#user-managed)
+   created when running the infrastructure template in your Google Cloud account. Aiven
+   [impersonates this service account](https://cloud.google.com/iam/docs/create-short-lived-credentials-direct)
+   and runs operations, such as creating VMs for service nodes, in your BYOC account.
+   :::
+
+1. You deploy your custom cloud resources supplying the generated privilege-bearing service
+   account to the Aiven platform, which gives Aiven the permissions
+   to securely access your Google Cloud account, create resources, and manage them onward.
+1. You select projects that can use your new custom clouds for creating services.
```

Comment:
I wonder if we should explicitly say `Aiven projects` since Google also has the concept of `Project` for a user account?